spec/acceptance/networkd_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'systemd with manage_networkd true' do
```

Comment:
Should we skip this on EL8, since we don't install EPEL?

Also, OracleLinux 7 looks messed up.

Reply:
Implemented: it now installs EPEL